lib/logmsg/logmsg.c

```diff
@@ -1829,6 +1824,18 @@ log_msg_stats_global_init(void)
   stats_unlock();
 }
 
+void
+log_msg_global_init(void)
+{
+  log_msg_registry_init();
+
+  /* NOTE: we always initialize counters as they are on stats-level(0),
+   * however we need to defer that as the stats subsystem may not be
+   * operational yet */
+  register_application_hook(AH_RUNNING, (ApplicationHookFunc) log_msg_register_stats, NULL);
```

Comment:
We agreed that this could be nice in another PR, but it needs additional work to unregister previously registered stats_counters.
Example is `log_tags_reinit_stats`:
```
      if (stats_check_level(3))
        stats_register_counter(3, &sc_key, SC_TYPE_PROCESSED, &log_tags_list[id].counter);
      else
        stats_unregister_counter(&sc_key, SC_TYPE_PROCESSED, &log_tags_list[id].counter);
```